### PR TITLE
Renamed dish model and controller to product

### PR DESCRIPTION
**Pull request by magnus-thor**
### https://www.pivotaltracker.com/story/show/152708933 ###
-------------------------
- Rename dish model to product
- Rename dish controller to product

### CODE REVIEW

**Review by olofvi — COMMENTED** [review]
Looks good to me :)

**Comment by BasilMawejje** [comment]
@tochman looks okay. Do you want to take a look too? I hope we didn't miss anything this time

**Review by tochman — APPROVED** [review]
Are the extra attribute for type of product, the conditional validation, and association to Protein class coming in another PR? If yes, this is ready to go.

File: db/schema.rb [hunk 1]
```diff
@@ -24,14 +24,6 @@
     t.string "image"
   end
 
-  create_table "salads", force: :cascade do |t|
-    t.string "name"
-    t.integer "price"
-    t.string "image"
-    t.datetime "created_at", null: false
-    t.datetime "updated_at", null: false
```
Comment (tochman) [inline db/schema.rb]: You never have to edit the schema directly. Just run the migrations. Not a blocker though.

**Comment by magnus-thor** [comment]
@tochman yes that is coming in a later PR, this is just about renaming the existing class and controller

**Pull request merged**